.github/workflows/ci.yml

```diff
@@ -636,6 +635,17 @@ jobs:
           && ! (contains(matrix.modules, 'trino-redshift') && contains(matrix.profile, 'fte-tests'))
         run: $MAVEN test ${MAVEN_TEST} -pl ${{ matrix.modules }} ${{ matrix.profile != '' && format('-P {0}', matrix.profile) || '' }}
       # Additional tests for selected modules
+      - name: S3 FileSystem Cloud Tests
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.TRINO_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.TRINO_AWS_SECRET_ACCESS_KEY }}
+          AWS_REGION: us-east-2
+          S3_BUCKET: trino-ci-s3fs-test
```

Comment:
Also it seems the bucket needs to be completely empty? I tried to run this test locally to debug some failures and it failed.

Can we either change tests to not depend on empty bucket or to at-least document what's required to run tests in the test itself?

Reply:
I was following the pattern of all the existing tests. The bucket needs to be empty because the FS tests verify listing the root. I couldn’t find a good way around this, but it’s way enough to create a new bucket for local testing. This hopefully won’t be an issue for the CI given that the test is fast and we’re unlikely to run two instances at once.

Reply:
> I was following the pattern of all the existing tests regarding the environment variable.

i agree with need to fix more than just this one place

> This hopefully won’t be an issue for the CI given that the test is fast and we’re unlikely to run two instances at once.

Does the test retry in case of  failure due to concurrency? do we have `@Flaky` annotation?